### PR TITLE
add custom jl_f_getfiled

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

File: src/compiler.jl [hunk 1]
```diff
@@ -430,6 +430,11 @@ function runtime_generic_augfwd(ret_ptr::Ptr{Any}, fn::Any, arg_ptr::Ptr{Any}, s
     tt = Tuple{map(x->eltype(Core.Typeof(x)), args)...}
     rt = Core.Compiler.return_type(fn, tt)
     annotation = guess_activity(rt)
+    if rt == Any
+        annotation = Duplicated{Any}
+    end
+    @show rt, tt, annotation, fn
```
Comment (wsmoses) [inline src/compiler.jl:436]: remove prints?

File: src/compiler.jl [hunk 2]
```diff
@@ -1960,7 +1965,7 @@ function jl_array_grow_end_rev(B::LLVM.API.LLVMBuilderRef, OrigCI::LLVM.API.LLVM
     return nothing
 end
 
-function setfield_fwd(B::LLVM.API.LLVMBuilderRef, OrigCI::LLVM.API.LLVMValueRef, gutils::API.EnzymeGradientUtilsRef, normalR::Ptr{LLVM.API.LLVMValueRef}, shadowR::Ptr{LLVM.API.LLVMValueRef}, tapeR::Ptr{LLVM.API.LLVMValueRef})::Cvoid
+function setfield_augfwd(B::LLVM.API.LLVMBuilderRef, OrigCI::LLVM.API.LLVMValueRef, gutils::API.EnzymeGradientUtilsRef, normalR::Ptr{LLVM.API.LLVMValueRef}, shadowR::Ptr{LLVM.API.LLVMValueRef}, tapeR::Ptr{LLVM.API.LLVMValueRef})::Cvoid
```
Comment (wsmoses) [inline src/compiler.jl:1968]: add a regular forward mode version?

**Comment by wsmoses** [comment]
@vchuravy add tests and rebase?

**Comment by vchuravy** [comment]
> @vchuravy add tests and rebase?

You ghost wrote this code if I recall

**Comment by wsmoses** [comment]
Closing in favor of https://github.com/EnzymeAD/Enzyme.jl/pull/390

**Pull request closed without merging**